doc/rbd/rbd-exclusive-locks.rst

```diff
@@ -60,6 +60,10 @@ Exclusive locking is mostly transparent to the user:
    exclusive lock. This is exposed by the ``--exclusive`` option for ``rbd
    device map`` command.
 
+.. note::
+   The ``exclusive-lock`` feature is incompatible with RBD advisory locks
+   (``rbd lock add`` and ``rbd lock rm`` commands).
```

Comment:
s/``rbd lock add``/the ``rbd lock add``/

This is one of these tricky situations where the article is used in English, but the only way to explain why is to stand upon fingerspitzengefühl.

Reply:
Anyway, this isn't a showstopper. I'm approving it. Does this get backported to S, R, and Q?

Reply:
> s/`rbd lock add`/the `rbd lock add`/

Done.

> Does this get backported to S, R, and Q?

Yes, to all supported releases.